plugins/notifier.js

```diff
@@ -0,0 +1,7 @@
+export default ({ _, store }, inject) => {
+  inject('notifier', {
```

Comment:
I'm not mega thrilled by the name `notifier`, but I found it in a tutorial and I couldn't come up with anything better.

Reply:
Is fine by me, gets the job done!